### PR TITLE
Feature/automated a11y tests

### PR DESCRIPTION
**Pull request by simonschwartz**

### CODE REVIEW

**Comment by dominikwilkowski** [comment]
It's a bit hard to review and I can't pull because of an merge conflict. Shall we manually reapply this in another branch? That would make it cleaner. Thoughts @simonschwartz?

File: packages/header/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 > Headers provide a consistent start to pages.
 
 
-## Content
+## Contentss
```
Comment (elisechant) [inline packages/header/README.md]: typo?

File: scripts/helper.js [hunk 2]
```diff
@@ -638,6 +638,8 @@ HELPER.init = () => {
 
 HELPER.init();
```
Comment (elisechant) [inline scripts/helper.js]: why init() here? is Helper a Singleton?

File: packages/header/README.md (hunk 1 above)
Comment (dominikwilkowski) [inline packages/header/README.md]: good catch. Will fix in my PR @simonschwartz

File: scripts/helper.js (hunk 2 above)
Comment (dominikwilkowski) [inline scripts/helper.js]: jup, CLI tool that listens to the arguments passed

**Comment by dominikwilkowski** [comment]
@elisechant he 75 file changes came from a budged merge with `master`. All those changes have been made in other PRs. as I said to clean this up I suggest we make a new branch and make the actual changes manually :)

File: packages/header/README.md (hunk 1 above)
Comment (dominikwilkowski) [inline packages/header/README.md]: fixed in #131

**Comment by dominikwilkowski** [comment]
see https://github.com/govau/uikit/tree/feature/automated-a11y-tests2

**Comment by simonschwartz** [comment]
Closing this PR - I messed up the rebase and will be easier to copy the real file changes into a new branch and raise a separate PR

**Pull request closed without merging**